datamigration.py

```diff
@@ -7,6 +7,8 @@
 from fabric.api import local, prompt  # pylint: disable=ungrouped-imports
 from fabric.contrib.console import confirm
 
+from os import path
+
```

Comment:
As a tiny matter of personal preference I prefer `import os.path` and then using `os.path.join`. I cannot remember the reasoning for this preference.